x-pack/plugins/security_solution/server/lib/siem_migrations/rules/task/agent/sub_graphs/translate_rule/nodes/translate_rule/prompts.ts

```diff
@@ -5,9 +5,13 @@
  * 2.0.
  */
 
-import type { MigrateRuleState } from '../../../types';
+import type { TranslateRuleState } from '../../types';
 
-export const getEsqlTranslationPrompt = (state: MigrateRuleState, query: string): string => {
+export const getEsqlTranslationPrompt = (
+  state: TranslateRuleState,
+  query: string,
```

Comment:
Shouldn't we use `inline_query` from the state? I think this query parameter can be removed